cryptor/cryptor.go

```diff
@@ -396,9 +396,9 @@ func (encrypted *EncryptedData) wrapKey(records *passvault.Records, clearKey []b
 // unwrapKey decrypts first key in keys whose encryption keys are in keycache
 func (encrypted *EncryptedData) unwrapKey(cache *keycache.Cache, user string) (unwrappedKey []byte, names []string, err error) {
 	var (
-		keyFound  error
-		fullMatch bool = false
-		nameSet        = map[string]bool{}
+		decryptErr error
```

Comment:
gofmt did this. The types and the initializers seem correctly aligned, although the gap with nameSet looks strange